example-secret/package.yaml

```
@@ -0,0 +1,16 @@
+name: Secret Greeter
+
+description: Action package description
```

Something like this maybe
```suggestion
description: Greets user with a secret message
```

---

Great catch, amended!